src/components/ebay-filter/index.marko

```diff
@@ -2,20 +2,22 @@ import { processHtmlAttributes } from "../../common/html-attributes"
 
 static var ignoredAttributes = [
     "selected",
-    "a11ySelectedText"
+    "a11ySelectedText",
+    "useAriaPressed"
 ];
 
 $ var baseClass = input.href ? "filter-link" : "filter-button";
+$ var a11yAttributes = input.useAriaPressed ? { "aria-pressed": (!input.href && state.selected && "true") } : "";
```

Comment:
We should make it like this:
`aria-pressed=(input.userAriaPressed !== false && !input.href && state.selected && "true")`

I'd also not put it as a variable and just pass it directly in the component.